include/cantera/oneD/StFlow.h

```diff
@@ -292,21 +280,20 @@ class StFlow : public Domain1D
     }
 
     /**
-     *  Evaluate the residual function for axisymmetric stagnation flow. If
-     *  j == npos, the residual function is evaluated at all grid points.
-     *  Otherwise, the residual function is only evaluated at grid points
-     *  j-1, j, and j+1. This option is used to efficiently evaluate the
-     *  Jacobian numerically.
+     * Evaluate the residual functions for axisymmetric stagnation flow. If j == npos,
+     * the residual function is evaluated at all grid points. Otherwise, the residual
+     * function is only evaluated at grid points j-1, j, and j+1. This option is used
+     * to efficiently evaluate the Jacobian numerically.
+     *
+     * These residuals at all the boundary grid points are evaluated using a default
+     * boundary condition that may be modified by a boundary object that is attached
+     * to the domain.The boundary object connected will modify these equations by
+     * subtracting the boundary object's values for V, T, mdot, etc. As a result,
+     * these residual equations will force the solution variables to the values of
+     * the connected boundary object.
      */
     void eval(size_t j, double* x, double* r, integer* mask, double rdt) override;
```

Comment:
I believe you changed the variable names in the implementation file. While they weren't consistent between `*.h` and `*.cpp` before either, please update so things match.

Reply:
The original name was diagg and I swapped to diag_global because, it's a clearer variable name. The original StFlow.h header has the mask name and the StFlow.cpp had the diagg name. Do we want to have the StFlow.h as diag_global? This also raises the issue that the other variable names in the header are not the same as those in the .cpp file for that function.

I was thinking of updating them in the header to match the eval() method in the .cpp file if that is ok.

Reply:
Names don't matter too much for me, but they should be consistent. @speth may have some input here ... it's better to have consensus first before having to retrace.